### PR TITLE
Review all tables and unify the way of data displaying. #602

### PR DESCRIPTION
**Pull request by MBovtriukProvectus**
Screenshot 2021-07-06 at 15 15 24
Screenshot 2021-07-06 at 15 12 13
Screenshot 2021-07-06 at 15 12 06

### CODE REVIEW

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx [hunk 1]
```diff
@@ -72,36 +72,44 @@ const Overview: React.FC<Props> = ({
...
-                        </span>
-                      }
-                      right
-                    >
-                      <DropdownItem
-                        onClick={() =>
-                          clearTopicMessages(clusterName, topicName, [
-                            partition,
-                          ])
+            {partitions?.length !== 0 && partitions?.length !== undefined ? (
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx:75]:
```suggestion
            {partitions?.length > 0 ? (
```

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx [hunk 2]
```diff
@@ -72,36 +72,44 @@ const Overview: React.FC<Props> = ({
...
+                          </span>
+                        </DropdownItem>
+                      </Dropdown>
+                    ) : null}
+                  </td>
+                </tr>
+              ))
+            ) : (
+              <tr>
+                <td colSpan={10}>No active Overview</td>
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx:110]:
```suggestion
                <td colSpan={10}>No Partitions found</td>
```

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Settings/Settings.tsx [hunk 3]
```diff
@@ -56,9 +56,15 @@ const Settings: React.FC<Props> = ({
...
-          ))}
+          {config.length > 0 ? (
+            config.map((item) => (
+              <ConfigListItem key={item.name} config={item} />
+            ))
+          ) : (
+            <tr>
+              <td colSpan={10}>No active Settings</td>
+            </tr>
+          )}
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Settings/Settings.tsx:67]: I think we can skip this table

File: kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx [hunk 4]
```diff
@@ -97,13 +97,19 @@ const Details: React.FC<Props> = ({
...
+                    key={consumer.consumerId}
+                    clusterName={clusterName}
+                    consumer={consumer}
+                  />
+                ))
+              ) : (
+                <tr>
+                  <td colSpan={10}>No active consumer group</td>
+                </tr>
+              )}
```
Comment (workshur) [inline kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx:112]: there is no need to check length for map.
```
<tbody>
  {items.map((consumer) => (
                  <ListItem
                    key={consumer.consumerId}
                    clusterName={clusterName}
                    consumer={consumer}
                  />
                ))}
   {{items.length === 0 && (
     <tr>
                  <td colSpan={10}>No active consumer group</td>
                </tr>
  )}

```

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx (hunk 2 above)
Comment (MBovtriukProvectus) [reply kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx:110]: done

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Settings/Settings.tsx (hunk 3 above)
Comment (MBovtriukProvectus) [reply kafka-ui-react-app/src/components/Topics/Topic/Details/Settings/Settings.tsx:67]: done

File: kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx (hunk 4 above)
Comment (MBovtriukProvectus) [reply kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx:112]: done

Comment (workshur) [reply kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx:112]: I see it is not done

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx (hunk 1 above)
Comment (MBovtriukProvectus) [reply kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx]: added partitions && partitions?.length > 0 becouse partitions can be undefined

Comment (workshur) [reply kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx]: `partitions?.length > 0` should help

**Review by workshur — CHANGES_REQUESTED** [review]
Pls cleanup commit history. This PR should not contain java code

**Pull request closed without merging**